DigitalLearningSolutions.Web/Views/MyAccount/Index.cshtml

```diff
@@ -0,0 +1,67 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.MyAccount

```

Comment:
Really minor thing - we usually add screenshots to the MR to make it easier to visualise the changes (just FYI for the next time).
I think one thing I'd change is removing the bottom border from the last row in the summary list to match the wireframes. I would also add a little bit of margin under the main title of the page.